### ISSUE TITLE
False positive in unnecessary_chaining when comments occur

### ISSUE THREAD

**Issue by jan-jezek** [abaplint/abaplint#2286]
```abap
TYPES: BEGIN OF t_test,
          " comment will break this check
           value  TYPE abap_bool,
         END OF t_test.
```

Check will fire in this case, but it should not. The check works correctly if no comment is present.

**Comment by larshp** [comment]
@jan-jezek try again, 2.82.15

**Comment by larshp** [comment]
closing
